README-dev.md

```diff
@@ -1,8 +1,8 @@
 # How to develop SnarkyJS
 
-This README collects information helpful for SnarkyJS core contributors.
+This README includes information that is helpful for SnarkyJS core contributors.
 
-## Run examples in Node
+## Run examples in a node
```

Comment:
That should actually be reverted to "in node", because we mean node, the JS runtime.

Reply:
`Run examples using NodeJS`

`in a node` would be interpreted as a Mina node